src/config/sled.rs

```diff
@@ -170,6 +173,28 @@ impl ConfigStore for SledConfigStore {
     }
 }
 
+impl ContactsStore for SledConfigStore {
+    fn save_contacts(&mut self, contacts: &[Contact]) -> Result<(), Error> {
+        // TODO: we probably want to store the contacts in a map of UUID > data
```

Comment:
`Uuid` is `Ord`, so I guess you could return a sorted Vec and let users use the binary search. This would require Ord on `ServiceAddress` though.

Reply:
But [`PhoneNumber`](https://docs.rs/phonenumber/0.3.1+8.12.9/phonenumber/struct.PhoneNumber.html) sadly isn't